website/client-old/js/services/costumeServices.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Costume', costumeFactory);
+
+  costumeFactory.$inject = [
+    'Content'
+  ];
+
+  function costumeFactory(Content) {
+    
+    function formatAnimal(name, type) {
+      if(type === 'pet') {
+        if(Content.petsInfo.hasOwnProperty(name)) {
+          return Content.petsInfo[name].text();
+        } else {
+          return window.env.t('noActivePet');
+        }
+      } else if(type === 'mount') {
+        if(Content.mountsInfo.hasOwnProperty(name)) {
+          return Content.mountsInfo[name].text();
+        } else {
+          return window.env.t('noActiveMount');
+        }
+      }
+    }
+
+    function formatBackground(background) {
+      var bg = Content.background;
```

Comment:
`background` does not exist. on the Content object. Only `backgrounds`, which is an object of the background sets.